modules/ROOT/pages/quick-start.adoc

```diff
@@ -8,6 +8,17 @@ Anypoint Monitoring provides data on Mule apps that are running on either CloudH
 [NOTE]
 Functional Monitoring is covered in xref:api-functional-monitoring::index.adoc[API Functional Monitoring].
 
+== Runtime Version Requirements
+
+For complete functionality, you can use the following runtime versions with Anypoint Monitoring:
+
+* Mule 3: versions 3.8.7 or newer
+* Mule 4: versions 4.1.6 or newer
+
+If you are using a Mule 4 version that is older than version 4.1.5, you cannot use built-in API metrics.
```

Comment:
Instead of using "built-in API metrics", let's use the the official name that we use in the rest of the doc - "Enhanced metrics for API Analytics".

Reply:
```suggestion
If you are using a Mule 4 version that is older than version 4.1.5, you cannot use enhanced metrics for API Analytics.
```